docs/write_test/write.md

```diff
@@ -0,0 +1,46 @@
+# Info for developers
+
+- [Basic concepts](concepts.md)
+  - [Test case and Test suite](concepts.md#test-case-and-test-suite)
+  - [Node and Environment](concepts.md#node-and-environment)
+  - [Extension](concepts.md#extension)
+    - [Platform](concepts.md#platform)
+    - [Tool and Script](concepts.md#tool-and-script)
+    - [Feature](concepts.md#feature)
+    - [Combinator](concepts.md#combinator)
+      - [Grid combinator](concepts.md#grid-combinator)
+      - [Batch combinator](concepts.md#batch-combinator)
+    - [Transformer](concepts.md#transformer)
+  - [Runbook](concepts.md#runbook)
+  - [Requirement and Capability](concepts.md#requirement-and-capability)
+- [Quick setup](setup.md)
```

Comment:
How about call it like development environment setup, or a shorter name? The "quick" is ambiguous.

Reply:
Changed to "Development setup".